bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/library/types/RawType.java

```diff
@@ -22,32 +22,45 @@
  */
 public class RawType implements PrimitiveType, State {
 
+    private static final String DEFAULT_MIME_TYPE = "image/jpeg";

```

Comment:
If there is a potential need for moving generic MIME-typed blocks of data as item states, its name is probably fine as is.  But I don't think it's a good idea to have item states that are untyped, generic blobs that don't have semantics for some type (like all the other state types -- OnOff, Percent, Decimal, Point, etc.).  

Doing so turns the state into just data instead of an object, and there may also be more natural transport formats and protocols.  Example: you can't stream a surveillance camera this way, and you obviously don't want to hold its full contents in memory in an item state.  So the model currently used by RawType won't be able to provide practical usage of a range of other resource types.  The range of relevant things that are commonly rendered by client components and are addressable via HTTP URL is very wide and won't all fit in a RawType.  

Something like a `URIType` seems more appropriate, that can update a `LinkItem`, and `LinkItems` can be associated with various UI widgets; for example, show an image on every update, or send `URIType` commands to change radio stations, switch surveillance camera feeds, etc.  This lets more natural protocols and renderers do all the work.

That is my thinking of why it should be renamed `ImageType`, or preferably, dropped as a state type and a more natural solution considered,